File: benchmark/results/ts-multilang/run-003/b-baseline/output/src/domain/ordering/services/OrderValidator.ts
import { Order } from "../aggregates/Order";

/** OrderValidator domain service for the ordering domain. */
export class OrderValidator {
  create(input: Partial<Order>): Order {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Order | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Order[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/ordering/services/OrderValidator.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** Domain service that validates orders before mutation. */
public class OrderValidator {
    /** Validate and create an order from the provided partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Validate by id. Returns null when not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} validated orders. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Validate a delete request for the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
